identity/src/main/java/androidx/security/identity/GattServer.java

```diff
@@ -404,6 +529,11 @@ void reportTransportSpecificSessionTermination() {
     }
 
     public void sendTransportSpecificTermination() {
+        if (supportL2CAP && mSocket != null && mSocket.isConnected()) {
+            // L2CAP doesn't read characteristics
+            // Send message to close connection status '20'
+            sendMessage(Util.fromHex("a16673746174757314"));
```

Comment:
suggest a comment for `a16673746174757314` instead of hard-coding literal CBOR in two spots